R/generate_template.R

```diff
@@ -12,6 +12,7 @@
 #' @param name name for the new application and directory
 #' @param location base path for creation of \code{name}
 #' @param sampleapp whether to create a sample shiny application
+#' @param dashboard_plus whether to include functionality from shinydashboardPlus, such as a right-sidebar
```

Comment:
I think we should rename this parameter - it only pertains to the right sidebar at this point.  if we made it something like right_sidebar and allowed assigning the icon as the name to enable the right sidebar with that icon - for example right_sidebar by default is NULL (off), or right_sidebar="gear" (ON, gear icon), right_sidebar="cog", etc.

Reply:
make sense indeed since it's only the right sidebar. I think the problem with just one parameter is, is that he user might (accidentally) specify a non-existing icon and this leads to a right sidebar that is not visible at all. I don't know how we can check that an icon exists. Maybe we need 2 params, one boolean to show/hide the right sidebar and another one for the icon?

Reply:
I think a note in the documentation should suffice that if the icon is incorrect the sidebar cannot be accessed, I don't think we need 2 parameters.

Reply:
Closing this comment, others cover the documentation/etc.